libs/blocks/adobetv/adobetv.js

```diff
@@ -1,19 +1,79 @@
 import { decorateAnchorVideo } from '../../utils/decorate.js';
-import { createTag } from '../../utils/utils.js';
+import { getConfig, createTag } from '../../utils/utils.js';
+
+const CaptionsLangMap = {
+  eng: [
+    'ae_ar', 'ae_en', 'africa', 'au', 'be_en', 'bg', 'ca', 'cz', 'dk', 'ee', 'gr_en',
+    'hk_en', 'hu', 'id_en', 'id_id', 'ie', 'il_en', 'il_he', 'in', 'lt', 'lu_en',
+    'lv', 'mena_ar', 'mena_en', 'my_en', 'my_ms', 'no', 'nz', 'ph_en', 'ph_fil',
+    'pl', 'ro', 'ru', 'sa_ar', 'sa_en', 'sg', 'si', 'sk', 'th_en', 'tr', 'ua', 'uk',
+    'vn_en', 'vn_vi', 'fi',
+  ],
+  fre_fr: ['be_fr', 'ch_fr', 'fr', 'lu_fr', 'ca_fr'],
+  ger: ['at', 'ch_de', 'lu_de', 'de'],
+  jpn: ['jp'],
+  ita: ['it', 'ch_it'],
+  spa: ['es'],
+  por_br: ['br', 'pt'],
+  tha: ['th_th'],
+  spa_la: ['ar', 'cl', 'co', 'la', 'mx', 'pe'],
+  dut: ['nl', 'be_nl'],
+  swe: ['se'],
+  chi_hans: ['cn'],
+  chi_hant: ['hk', 'tw'],
+  hin: ['in_hi'],
+  kor: ['kr'],
+};
+
+/**
+ * Maps a video URL's captions parameter to the appropriate language code based on geo location.
+ *
+ * @param {string} videoUrl - The URL of the Adobe TV video
+ * @param {string} geo - The geographic location code (e.g. 'fr', 'jp', 'de')
+ * @returns {string} The updated video URL with the correct captions language parameter
+*
+ * @example
+ * // Updates captions to French for French geo
+ * updateCaptionsLang('https://video.tv.adobe.com/v/123456?captions=eng', 'fr')
+ * // Returns: 'https://video.tv.adobe.com/v/123456?captions=fre_fr'
+ *
+ * @example
+ * // Preserves URL if captions param not present
+ * updateCaptionsLang('https://video.tv.adobe.com/v/123456', 'fr')
+ * // Returns: 'https://video.tv.adobe.com/v/123456'
+ */
```

Comment:
Because there's no build to remove comments/minimize code/etc., we try to avoid JSDoc comments, since these add weight to the end file, without adding customer value. This method is straightforward enough to not need all the documentation.
```suggestion

```